app/src/main/java/com/internship/move/feature/splash/SplashFragment.kt

```diff
@@ -1,24 +1,55 @@
 package com.internship.move.feature.splash
 
+import android.content.Context
 import android.os.Bundle
 import android.os.Handler
 import android.os.Looper
 import android.view.View
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
 import com.internship.move.R
+import com.internship.move.feature.authentication.RegisterFragmentDirections
+import com.internship.move.feature.onboarding.OnboardingFragmentDirections
 
 class SplashFragment : Fragment(R.layout.fragment_splash) {
 
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
-
-        Handler(Looper.getMainLooper()).postDelayed({
+    private val myHandler: Handler = Handler(Looper.getMainLooper())
```

Comment:
The naming of these two variables doesn't tell us much about what they do
I'd suggest something like `navigationHandler` and `navigationRunnableJob` to make them more expressive at first glance